kitsune/search/v2/search.py

```diff
@@ -198,5 +198,5 @@ def get_filter(self):
     def make_result(self, hit):
         index = hit.meta.index
         for child in self._children:
-            if index == child.get_index():
+            if index.split("_")[:-1] == child.get_index().split("_")[:-1]:
```

Comment:
nit: why not split("_")[0]? Just to make it more readable

Reply:
With the prefix included (`sumo_wikidocument_20210210`) that wouldn't work - I considered creating a function for this, it'll use a few more chars but probably worth it for the readability, I'll see.